.github/workflows/nightly.yml

```diff
@@ -17,7 +17,43 @@ on:
 
 jobs:
 
-    BUILD-TEST:
+    PYTHON-3-8:
+        uses: ./.github/workflows/build-test.yml
+        with:
+            wf_category: NIGHTLY
+            python: 3.8.17
+            gitref: ${{ github.ref }}
+
+            test_label_solo: aws-avx2-32G-a10g-24G
+            test_label_multi: aws-avx2-192G-4-a10g-96G
+            test_timeout: 480
+            test_skip_list: neuralmagic/tests/skip-for-remote-push-tmp.txt
```

Comment:
Again, the py310 job is not using the `-tmp` version of this file.

Reply:
same comment as above.